### PR TITLE
KK-966 | Update Ticketmaster texts and translations

### PR DESCRIPTION
**Pull request by tuomas777**
## Context

[KK-966](https://helsinkisolutionoffice.atlassian.net/browse/KK-966)

### CODE REVIEW

File: src/common/translation/i18n/en.json [hunk 1]
```diff
@@ -534,6 +528,7 @@
     },
     "text": "The page you are looking for couldn't be found"
   },
+  "openInExternalDomainAriaLabel": "Accessibility Statement",
```
Comment (nikomakela) [inline src/common/translation/i18n/en.json:531]: is this right?

File: src/common/translation/i18n/fi.json [hunk 2]
```diff
@@ -534,6 +528,7 @@
     },
     "text": "Etsimääsi sivua ei löytynyt"
   },
+  "openInExternalDomainAriaLabel": "Saavutettavuusseloste",
```
Comment (nikomakela) [inline src/common/translation/i18n/fi.json:531]: Is this right?

File: src/common/translation/i18n/sv.json [hunk 3]
```diff
@@ -534,6 +528,7 @@
     },
     "text": "Sidan du söker kunde inte hittas"
   },
+  "openInExternalDomainAriaLabel": "Tillgänglighetsredogörelse",
```
Comment (nikomakela) [inline src/common/translation/i18n/sv.json:531]: is this right?

**Review by nikomakela — APPROVED** [review]
1 strange translation key-value pair

**Comment by tuomas777** [comment]
> 1 strange translation key-value pair

Yeah, that was some leftover. Well spotted!

**Pull request merged**